### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by vbevziuk**
- [DEMO LINK](https://vbevziuk.github.io/layout_calendar/)
- [TEST REPORT LINK](https://vbevziuk.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss
Comment (maxim2310) [inline src/styles/main.scss]: would be nice move it to reset file

File: src/index.html
Comment (maxim2310) [inline src/index.html]:
```suggestion
    <div class="calendar calendar--start-day--thu calendar--month-length--29">
```

File: src/index.html
Comment (maxim2310) [inline src/index.html]:
```suggestion
      <div class="calendar__day"></div>
```

File: src/styles/main.scss
Comment (maxim2310) [inline src/styles/main.scss]: block must havent external margin

File: src/styles/main.scss
Comment (maxim2310) [inline src/styles/main.scss]:
```suggestion
  width: (($squeare-size * 7) + 6);
```

File: src/styles/main.scss
Comment (maxim2310) [inline src/styles/main.scss]:
```suggestion
      margin-left: ($squeare-size + 1) * $number;
```

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple things needs to be fixed before approval)
and would be nice to center your calendar on page

File: src/styles/main.scss
Comment (maxim2310) [inline src/styles/main.scss]: would be better to center it using flex box to body

**Review by maxim2310 — APPROVED** [review]
Good job. Suggest some improvements.

File: src/styles/main.scss
Comment (vbevziuk) [reply src/styles/main.scss]: > would be better to center it using flex box to body

why can't I center this element vertically with `align-items: center;`?

body {
  display: flex;
  justify-content: center;
  align-items: center;
  margin: 0;
}

**Pull request closed without merging**
